DataCapturing.xcodeproj/xcshareddata/xcbaselines/3D0BDEA91FB1B951004EA25E.xcbaseline/0215D036-B6EF-4343-A287-D0A58E604DF7.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This and the following file are generated by Xcode during testing. It stores performance values captured during performance testing. Future performance tests are compared against these values.

Reply:
That's cool. (monitoring the code performance)